engines/order_management/app/controllers/order_management/reports/enterprise_fee_summaries_controller.rb

```diff
@@ -0,0 +1,64 @@
+module OrderManagement
+  module Reports
+    class EnterpriseFeeSummariesController < Spree::Admin::BaseController
```

Comment:
I changed this at the same time as I moved the file (should have been a separate commit): the namespace here was Spree::Admin::Reports